src/sparsezoo/inference/inference_runner.py

```diff
@@ -97,9 +100,6 @@ def generate_outputs(
                 f"Supported engines: {self.engine_type_to_iterator.keys()}."
             )
 
-        if save_to_tar:
-            self._save_outputs_to_tar(iterator, engine_type)
-
```

Comment:
Does this functionality exist somewhere else now?

Reply:
Yes. It lives now in `sparsezoo/src/sparsezoo/objects/utils.py :: save_outputs_to_tar`

Reply:
Does this method still need to call that function though? Is `save_to_tar` flag not supported anymore?